addons/binding/org.openhab.binding.vallox/src/main/java/org/openhab/binding/vallox/internal/serial/ValloxStore.java

```diff
@@ -0,0 +1,231 @@
+    public int dCFanOutputAdjustment; // VALLOX_VARIABLE_DC_FAN_OUTPUT_ADJUSTMENT
+    public int inputFanStopThreshold; // VALLOX_VARIABLE_INPUT_FAN_STOP
+    public int heatingSetPoint; // VALLOX_VARIABLE_HEATING_SET_POINT
+    public int preHeatingSetPoint; // VALLOX_VARIABLE_PRE_HEATING_SET_POINT
+    public int hrcBypassThreshold; // VALLOX_VARIABLE_HRC_BYPASS
+    public int cellDefrostingThreshold; // VALLOX_VARIABLE_CELL_DEFROSTING
+
+    // program
+    public int adjustmentIntervalMinutes; // VALLOX_VARIABLE_PROGRAM
+    public boolean automaticHumidityLevelSeekerState; // VALLOX_VARIABLE_PROGRAM
+    public boolean boostSwitchMode; // VALLOX_VARIABLE_PROGRAM
+    public boolean radiatorType; // VALLOX_VARIABLE_PROGRAM
+    public boolean cascadeAdjust; // VALLOX_VARIABLE_PROGRAM
+
+    // program2
+    public boolean maxSpeedLimitMode; // VALLOX_VARIABLE_PROGRAM2
+
+    public int serviceReminder; // VALLOX_VARIABLE_SERVICE_REMINDER
+
+    // ioport multi purpose 1
+    public boolean postHeatingOn; // VALLOX_VARIABLE_IOPORT_MULTI_PURPOSE_1
+
+    // ioport multi purpose 2
+    public boolean damperMotorPosition; // VALLOX_VARIABLE_IOPORT_MULTI_PURPOSE_2
+    public boolean faultSignalRelayClosed; // VALLOX_VARIABLE_IOPORT_MULTI_PURPOSE_2
+    public boolean supplyFanOff; // VALLOX_VARIABLE_IOPORT_MULTI_PURPOSE_2
+    public boolean preHeatingOn; // VALLOX_VARIABLE_IOPORT_MULTI_PURPOSE_2
+    public boolean exhaustFanOff; // VALLOX_VARIABLE_IOPORT_MULTI_PURPOSE_2
+    public boolean firePlaceBoosterClosed; // VALLOX_VARIABLE_IOPORT_MULTI_PURPOSE_2
+
+    public int incommingCurrent; // VALLOX_VARIABLE_CURRENT_INCOMMING
+    public int lastErrorNumber; // VALLOX_VARIABLE_LAST_ERROR_NUMBER
+
+    // those variables might be implemented in future
+    // VALLOX_VARIABLE_IOPORT_FANSPEED_RELAYS
+    // VALLOX_VARIABLE_INSTALLED_CO2_SENSORS
+    // VALLOX_VARIABLE_POST_HEATING_ON_COUNTER
+    // VALLOX_VARIABLE_POST_HEATING_OFF_TIME
+    // VALLOX_VARIABLE_POST_HEATING_TARGET_VALUE
+    // VALLOX_VARIABLE_FLAGS_1
+    // VALLOX_VARIABLE_FLAGS_2
+    // VALLOX_VARIABLE_FLAGS_3
+    // VALLOX_VARIABLE_FLAGS_4
+    // VALLOX_VARIABLE_FLAGS_5
+    // VALLOX_VARIABLE_FLAGS_6
+    // VALLOX_VARIABLE_FIRE_PLACE_BOOSTER_COUNTER
+    // VALLOX_VARIABLE_MAINTENANCE_MONTH_COUNTER
+
+    // calculated properties
+    public int inEfficiency;
+    public int outEfficiency;
+    public int averageEfficiency;
+
+    // virtual properties to be able to poll for this variable
+    public int selectStatus;
+    public int program; // VALLOX_VARIABLE_PROGRAM
+    public int program2; // VALLOX_VARIABLE_PROGRAM2
+    public int ioPortMultiPurpose1; // VALLOX_VARIABLE_IOPORT_MULTI_PURPOSE_1
+    public int ioPortMultiPurpose2; // VALLOX_VARIABLE_IOPORT_MULTI_PURPOSE_2
+
+    public boolean suspended = false;
+
+    @Override
+    public String toString() {
+        StringBuilder builder = new StringBuilder();
```

Comment:
The beauty of the StringBuilder is that you can chain it. Next time please generate it, safes you some work.

Reply:
I already used the Eclipse toString() generator. It has two styles to generate with the StringBuilder, either with or without chaining. I now changed it to chaining. It's a bit shorter.